### ISSUE TITLE
Building debian packages seems to be broken

### ISSUE THREAD

**Issue by coffeecup-winner** [mosra/magnum#237]
I've successfully built and installed a package around a month ago, but today I can't do the same following the same guide: http://doc.magnum.graphics/magnum/building.html#building-packages-deb. OS is Ubuntu 16.04.

I've tried both master top-of-tree and v2018.02 with the same results:
```
...
   dh_install
   dh_installdocs
   dh_installchangelogs
   dh_perl
   dh_link
   dh_strip_nondeterminism
   dh_compress
   dh_fixperms
   dh_strip
   dh_makeshlibs
   debian/rules override_dh_shlibdeps
make[1]: Entering directory '/home/coffeecup-winner/work/magnum/magnum'
dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
dpkg-shlibdeps: error: couldn't find library libMagnumAudio.so.1 needed by debian/magnum/usr/lib/magnum/audioimporters/WavAudioImporter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/bin/magnum-info (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumText.so.1 needed by debian/magnum/usr/lib/magnum/fonts/MagnumFont.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/lib/magnum/fonts/MagnumFont.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumMeshTools.so.1 needed by debian/magnum/usr/lib/magnum/importers/ObjImporter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/lib/magnum/importers/ObjImporter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumText.so.1 needed by debian/magnum/usr/lib/magnum/fontconverters/MagnumFontConverter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/lib/magnum/fontconverters/MagnumFontConverter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/bin/magnum-imageconverter (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumTextureTools.so.1 needed by debian/magnum/usr/bin/magnum-distancefieldconverter (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/bin/magnum-distancefieldconverter (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/lib/magnum/importers/TgaImporter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/lib/magnum/imageconverters/TgaImageConverter.so (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumText.so.1 needed by debian/magnum/usr/bin/magnum-fontconverter (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnum.so.1 needed by debian/magnum/usr/bin/magnum-fontconverter (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: couldn't find library libMagnumAudio.so.1 needed by debian/magnum/usr/bin/magnum-al-info (ELF format: 'elf64-x86-64'; RPATH: '')
dpkg-shlibdeps: error: cannot continue due to the errors listed above
Note: libraries are not searched in other binary packages that do not have any shlibs or symbols file.
To help dpkg-shlibdeps find private libraries, you might need to use -l.
dh_shlibdeps: dpkg-shlibdeps -Tdebian/magnum.substvars --ignore-missing-info debian/magnum/usr/bin/magnum-imageconverter debian/magnum/usr/bin/magnum-fontconverter debian/magnum/usr/bin/magnum-distancefieldconverter debian/magnum/usr/bin/magnum-al-info debian/magnum/usr/bin/magnum-info debian/magnum/usr/lib/magnum/importers/ObjImporter.so debian/magnum/usr/lib/magnum/importers/TgaImporter.so debian/magnum/usr/lib/magnum/fontconverters/MagnumFontConverter.so debian/magnum/usr/lib/magnum/fonts/MagnumFont.so debian/magnum/usr/lib/magnum/imageconverters/TgaImageConverter.so debian/magnum/usr/lib/magnum/audioimporters/WavAudioImporter.so returned exit code 2
debian/rules:42: recipe for target 'override_dh_shlibdeps' failed
make[1]: *** [override_dh_shlibdeps] Error 2
make[1]: Leaving directory '/home/coffeecup-winner/work/magnum/magnum'
debian/rules:14: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
```

Some (possibly) relevant folder contents:
```
$ ls -l debian/magnum/usr/lib
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   19 Mar 12 21:17 libMagnumAudio.so -> libMagnumAudio.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   24 Mar 12 21:17 libMagnumDebugTools.so -> libMagnumDebugTools.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   23 Mar 12 21:17 libMagnumMeshTools.so -> libMagnumMeshTools.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   24 Mar 12 21:17 libMagnumPrimitives.so -> libMagnumPrimitives.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   24 Mar 12 21:17 libMagnumSceneGraph.so -> libMagnumSceneGraph.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   21 Mar 12 21:17 libMagnumShaders.so -> libMagnumShaders.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   20 Mar 12 21:17 libMagnumShapes.so -> libMagnumShapes.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   14 Mar 12 21:17 libMagnum.so -> libMagnum.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   18 Mar 12 21:17 libMagnumText.so -> libMagnumText.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner   26 Mar 12 21:17 libMagnumTextureTools.so -> libMagnumTextureTools.so.1
drwxr-xr-x 7 coffeecup-winner coffeecup-winner 4096 Mar 12 21:17 magnum

$ ls -l debian/tmp/usr/lib
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       19 Mar 12 21:17 libMagnumAudio.so -> libMagnumAudio.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       21 Mar 12 21:17 libMagnumAudio.so.1 -> libMagnumAudio.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1132888 Mar 12 21:16 libMagnumAudio.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       24 Mar 12 21:17 libMagnumDebugTools.so -> libMagnumDebugTools.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       26 Mar 12 21:17 libMagnumDebugTools.so.1 -> libMagnumDebugTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  5950360 Mar 12 21:16 libMagnumDebugTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1258130 Mar 12 21:15 libMagnumGlxApplication.a
-rw-r--r-- 1 coffeecup-winner coffeecup-winner   451072 Mar 12 21:15 libMagnumGlxContext.a
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       23 Mar 12 21:17 libMagnumMeshTools.so -> libMagnumMeshTools.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       25 Mar 12 21:17 libMagnumMeshTools.so.1 -> libMagnumMeshTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1211920 Mar 12 21:16 libMagnumMeshTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner   480790 Mar 12 21:15 libMagnumOpenGLTester.a
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       24 Mar 12 21:17 libMagnumPrimitives.so -> libMagnumPrimitives.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       26 Mar 12 21:17 libMagnumPrimitives.so.1 -> libMagnumPrimitives.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  2328320 Mar 12 21:16 libMagnumPrimitives.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       24 Mar 12 21:17 libMagnumSceneGraph.so -> libMagnumSceneGraph.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       26 Mar 12 21:17 libMagnumSceneGraph.so.1 -> libMagnumSceneGraph.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  3200648 Mar 12 21:16 libMagnumSceneGraph.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1110446 Mar 12 21:15 libMagnumSdl2Application.a
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       21 Mar 12 21:17 libMagnumShaders.so -> libMagnumShaders.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       23 Mar 12 21:17 libMagnumShaders.so.1 -> libMagnumShaders.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1956040 Mar 12 21:16 libMagnumShaders.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       20 Mar 12 21:17 libMagnumShapes.so -> libMagnumShapes.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       22 Mar 12 21:17 libMagnumShapes.so.1 -> libMagnumShapes.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1306272 Mar 12 21:16 libMagnumShapes.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       14 Mar 12 21:17 libMagnum.so -> libMagnum.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       16 Mar 12 21:17 libMagnum.so.1 -> libMagnum.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner 11022440 Mar 12 21:15 libMagnum.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       18 Mar 12 21:17 libMagnumText.so -> libMagnumText.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       20 Mar 12 21:17 libMagnumText.so.1 -> libMagnumText.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner  1511888 Mar 12 21:16 libMagnumText.so.1.0
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       26 Mar 12 21:17 libMagnumTextureTools.so -> libMagnumTextureTools.so.1
lrwxrwxrwx 1 coffeecup-winner coffeecup-winner       28 Mar 12 21:17 libMagnumTextureTools.so.1 -> libMagnumTextureTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner   726424 Mar 12 21:16 libMagnumTextureTools.so.1.0
-rw-r--r-- 1 coffeecup-winner coffeecup-winner   782236 Mar 12 21:15 libMagnumWindowlessGlxApplication.a
drwxrwxr-x 7 coffeecup-winner coffeecup-winner     4096 Mar 12 21:17 magnum
```

It looks like the libs it's searching for are in the `/tmp` subdirectory, while it's trying `/magnum`.

**Comment by coffeecup-winner** [comment]
Builds fine on commit `52613a2ad94e7c6f11c7898e162f636db7d07980` (Feb 1).

**Comment by mosra** [comment]
Oh, sorry about this. I've bumped SOVERSION numbers but the package glob files don't reflect that. Fixing now ...

**Comment by mosra** [comment]
Should be fixed in mosra/magnum@90fd2a365e59a561c9a87734c0297c8436ee5e45, mosra/magnum-integration@fd7035723ec2e8c54113eead1e4703c6a62f10c3 and mosra/magnum-extras@c3be8370acd8bd186a74e263c7160f5112f0c06f. The plugins repo wasn't affected by this.

*(Wow, it took me five hours?)*

**Comment by coffeecup-winner** [comment]
Thanks a lot, works great now!
